addons/website/views/snippets/s_company_team_basic.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_basic" name="Team Basic">
+    <section class="s_company_team_basic o_colored_level o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="10" style="column-gap: 32px;">
```

Comment:
I thin we can combine `column-gap: 32px;` and `--grid-item-padding-x: 16px;` into `--grid-item-padding-x: 32px;` (column gap disappears in mobile, that's why I think it's better to keep the padding)